### ISSUE TITLE
Hub is not reading Bert Model correctly

### ISSUE THREAD

**Issue by din1881** [tensorflow/hub#832]
I am using tensorflow hub to read the BERT model as follows:

`b_preprocess = hub.KerasLayer("https://tfhub.dev/tensorflow/bert_en_uncased_preprocess/3")`
`b_encoder = hub.KerasLayer("https://tfhub.dev/tensorflow/bert_en_uncased_L-12_H-768_A-12/4")`

But then when I try running the below code, it returns empty tensors, I am running the code on AzureML, also, it worth noting that I didn't have this problem before, and I am using the same exact code as before:

`text_test = ['The replacement monitor arrived today']`
`text_preprocessed = b_preprocess(text_test)`
`print(f'Keys       : {list(text_preprocessed.keys())}')`
`print(f'Shape      : {text_preprocessed["input_word_ids"].shape}')`
`print(f'Word Ids   : {text_preprocessed["input_word_ids"][0, :12]}')`
`print(f'Input Mask : {text_preprocessed["input_mask"][0, :12]}')`
`print(f'Type Ids   : {text_preprocessed["input_type_ids"][0, :12]}')`

Output:
`Keys       : ['input_word_ids', 'input_mask', 'input_type_ids']`
`Shape      : (1, 128)`
`Word Ids   : Tensor("strided_slice:0", shape=(12,), dtype=int32)`
`Input Mask : Tensor("strided_slice_1:0", shape=(12,), dtype=int32)`
`Type Ids   : Tensor("strided_slice_2:0", shape=(12,), dtype=int32)`

**Comment by sanatmpa1** [comment]
@din1881,

Can you share the complete standalone code to reproduce the issue from our end? Thanks!

**Comment by sanatmpa1** [comment]
@din1881,

Closing this issue due to inactivity, Please feel free to reopen this issue or create a new issue with more details if the problem still persists. Thanks!
